magicctapipe/scripts/lst1_magic/lst1_magic_dl2_to_dl3.py

```diff
@@ -471,38 +762,54 @@ def main():
     parser = argparse.ArgumentParser()
 
     parser.add_argument(
-        '--input-file-dl2', '-d', dest='input_file_dl2', type=str, required=True,
-        help='Path to an input DL2 data file.',
+        "--input-file-dl2",
+        "-d",
+        dest="input_file_dl2",
+        type=str,
+        required=True,
+        help="Path to an input DL2 data file.",
     )
 
     parser.add_argument(
-        '--input-file-irf', '-i', dest='input_file_irf', type=str, required=True,
-        help='Path to an input IRF file.',
+        "--input-dir-irf",
+        "-i",
+        dest="input_dir_irf",
+        type=str,
+        required=False,
```

Comment:
I set required to False before because I used two different options for single file and directory, and only one should be used, but since now the code used only -dir- option, I think it should be again required